documentation/modules/auxiliary/scanner/mssql/mssql_login.md

```diff
@@ -15,6 +15,41 @@ A docker container can be spun up with the following command to test this module
 
 ## Options
 
+### CreateSession
+
+When using the mssql_login module, the CreateSession option can be used to obtain an interactive
```

Comment:
Do you thing maybe having the full path here would be a good idea, my thinking is having the full path that would be displayed on console when using that module. It might be good for copy pasting for users, as if only one search result is found it will use the module automatically.
```suggestion
When using the `scanner/mssql/mssql_login` module, the CreateSession option can be used to obtain an interactive
```